### ISSUE TITLE
How can I train a model personalized to a particular player?

### ISSUE THREAD

**Issue by FrugoFruit90** [CSSLab/maia-chess#36]
Is there a script that would do pretty much the same as train_maia.py, but starting with a given set of weights?

This achieves the highest move-matching accuracy in the paper but I didn't find it mentioned in the readme.

**Comment by reidmcy** [comment]
This is just the code for the original (KDD 2020) paper. The personalize models paper has not been published yet so we don't have a code release.

**Comment by bradley-Amuj** [comment]
can I get a personalized model if I use training data that is from one particular play?

**Comment by reidmcy** [comment]
In theory yes, but we needed about 10 million games to get good performance when starting from scratch. So I don't think there's anyone with enough games for that to work.

**Comment by AlexMolas** [comment]
isn't it possible to use a pre-trained model and then fine-tune it with a small set of games from a particular user?

**Comment by reidmcy** [comment]
Yes, we're working on the code release for that right now: https://github.com/CSSLab/maia-individual

**Comment by FrugoFruit90** [comment]
Would love to use that feature too.
